NodeManager.cpp

```diff
@@ -1945,8 +2085,8 @@ int NodeManager::registerSensor(int sensor_type, int pin, int child_id) {
     else if (sensor_type == SENSOR_LATCHING_RELAY) return registerSensor(new SensorLatchingRelay(child_id, pin));
   #endif
   #if MODULE_DHT == 1
-    else if (sensor_type == SENSOR_DHT11 || sensor_type == SENSOR_DHT22) {
-      int dht_type = sensor_type == SENSOR_DHT11 ? DHT11 : DHT22;
+    else if (sensor_type == SENSOR_DHT11 || sensor_type == SENSOR_DHT22 || sensor_type == SENSOR_DHT21) {
+      int dht_type = sensor_type == SENSOR_DHT11 ? DHT21 : DHT22;
```

Comment:
Looking at https://github.com/adafruit/DHT-sensor-library/blob/master/DHT.h seems DHT11, DHT21 and DHT22 are defined differently so we probably need to adjust the dht_type to have three values as well